sdks/java/core/src/test/java/org/apache/beam/sdk/io/range/OffsetRangeTrackerTest.java

```diff
@@ -128,13 +139,13 @@ public void testGetPositionForFractionDense() throws Exception {
   @Test
```

Comment:
I'm surprised that the `getPositionForFraction` tests above didn't have to change after you removed the `+1`. Now that we're moved fractions slightly, we probably should also move the positions we infer from fractions, right?

Take a look and see if that is right.